content/docs/fundamentals/converting/resolving-errors@zh_CN.md

```diff
@@ -0,0 +1,350 @@
+```
+
+刷新网页并检查验证工具；您应该不会再看到任何错误！
+
+您现在已经获得了一篇有效的 AMP 文档，但图片的显示效果并不好，因为它在网页中的位置很不自然。默认情况下，当您为 `amp-img` 指定高度和宽度后，AMP 会将尺寸固定为您指定的值；但如果 AMP 能够对图片进行缩放，使其能够以*自适应的方式*进行拉伸并适应网页的布局，无论屏幕尺寸多大都没关系，那岂不是很棒吗？
+
+{{ image('/static/img/docs/tutorials/tut-convert-html-not-responsive.png', 412, 660, align='center third', caption="我们的图片不是自适应图片。") }}
+
+幸运的是，AMP 可以根据您指定的宽度和高度来确定元素的宽高比。这样一来，AMP 布局系统便可以按照多种方式放置和缩放元素。`layout` 属性会告知 AMP 您想要如何放置和缩放元素。
+
+我们将 layout 属性**设为** `responsive`，让图片能够缩放并调整大小：
+
+```html
+<amp-img src="mountains.jpg" layout="responsive" width="266" height="150"></amp-img>
+```
+
+瞧！图片采用了正确的宽高比，并以自适应的方式填满了屏幕的整个宽度。
+
+{{ image('/static/img/docs/tutorials/tut-convert-html-responsive.png', 412, 660, align='center third', caption="我们的图片现在是自适应图片了！") }}
+
+{% call callout('继续阅读', type='read') %}
+要详细了解 AMP 布局系统，请参阅 [AMP 布局规范](/zh_cn/docs/design/amp-html-layout.html)。
+{% endcall %}
+
+## 成功了！
+
+现在，您的 AMP 文档应该会大致如下所示：
+
+```html
+<!doctype html>
+<html ⚡ lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+
+    <link rel="canonical" href="/article.html">
+    <link rel="shortcut icon" href="amp_favicon.png">
+
+    <title>News Article</title>
+
+    <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
+    <style amp-custom>
+      body {
+        width: auto;
+        margin: 0;
+        padding: 0;
+      }
+
+      header {
+        background: Tomato;
+        color: white;
+        font-size: 2em;
+        text-align: center;
+      }
+
+      h1 {
+        margin: 0;
+        padding: 0.5em;
+        background: white;
+        box-shadow: 0px 3px 5px grey;
+      }
+
+      p {
+        padding: 0.5em;
+        margin: 0.5em;
+      }
+    </style>
+    <script async src="https://cdn.ampproject.org/v0.js"></script>
+  </head>
+  <body>
+    <header>
+      News Site
+    </header>
+    <article>
+      <h1>Article Name</h1>
+
+      <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Etiam egestas tortor sapien, non tristique ligula accumsan eu.</p>
+
+      <amp-img src="mountains.jpg" layout="responsive" width="266" height="150"></amp-img>
+    </article>
+  </body>
+</html>
+```
+
+刷新网页，看看控制台的输出结果。您应该会看到以下消息：
+
+<pre class="success-text">
```

Comment:
All of these code blocks need to be in EN, so please update with EN translation (https://www.ampproject.org/docs/fundamentals/converting/resolving-errors).